pkg/query/queryexecute/execute.go

```diff
@@ -27,7 +27,7 @@ func RunInteractiveSession(ctx context.Context, initData *query.InitData) {
 
 	// print the data as it comes
 	for r := range resultsStreamer.Results {
-		display.ShowOutput(ctx, r)
+		display.ShowOutput(ctx, r, display.ShowTimingIfEnabled())
```

Comment:
ShowTimingIfEnabled is a bit counterintuittive - why do we need an option for this - can we not just.... show it if it;s enabled

It would feel more intuitive to have a `diusable` timing option

Reply:
`ShowTimingIfEnabled` peeks into the `viper` config and only displays the timing data if it is enabled in `viper`

This is meant to cover `.timing` commands in the interactive prompt